### DISCUSSION TITLE
Bug where Craft Control Panel Admin session does the  "Your session has ended. Enter your password to log back in" javascript popup after exactly one minute

### DISCUSSION THREAD

**Discussion by matthewstick** [craftcms/cms#15126] [General]
In the control panel, I'm getting the "Your session has ended. Enter your password to log back in." popup after exactly one minute.

The popup actually does nothing, because if I refresh the page I'm still logged in.

I've tried a couple settings to try to bypass this, but nothign is working:
->enableCsrfProtection(false)
->userSessionDuration(24 * 60 * 60) // 1 day
->requireMatchingUserAgentForSession(false)

Note we are using php sessions.  But I've enabled and disabled php sessions and it doesn't matter.  Still has the issue.

Here is version info:
PHP version | 8.2.7
Linux 5.14.0-284.30.1.el9_2.x86_64
MySQL 8.0.28
Craft Pro 4.9.4

Here is the video:
https://github.com/craftcms/cms/assets/89614/786af02e-2e0e-4790-a1b7-29a1a6efb662

**Comment by matthewstick** [comment]
Turns out some cloud flare caching settings were causing the pages to think they were no longer active?  Got around this by bypassing caching differently.  Feel free to close or if you think there is something here then take as you like.  Hope this helps someone

**Reply by brandonkelly** [reply]
Glad you got it sorted!
